osu.Game/Online/Placeholders/LoginPlaceholder.cs

```diff
@@ -14,15 +14,15 @@ public sealed class LoginPlaceholder : Placeholder
         [Resolved(CanBeNull = true)]
         private LoginOverlay login { get; set; }
 
-        public LoginPlaceholder()
+        public LoginPlaceholder(string actionMessage)
         {
             AddIcon(FontAwesome.Solid.UserLock, cp =>
             {
                 cp.Font = cp.Font.With(size: TEXT_SIZE);
                 cp.Padding = new MarginPadding { Right = 10 };
             });
 
-            AddText(@"Please sign in to view online leaderboards!");
+            AddText(actionMessage);
```

Comment:
I meant the concatenation would reside here:
`AddText($"Please sign in to {actionMessage}!");`

Reply:
This will not localise well. Fine as-is.